packages/ios_platform_images/CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
+## 0.0.2
```

Comment:
Should be 0.1.0 since you're removing a class.

Reply:
Done.